app/core/constants.js

```diff
@@ -120,6 +120,8 @@ export const DEFAULT_WALLET = {
 
 export const DEFAULT_CURRENCY_CODE = 'usd'
 
+export const DEFAULT_THEME = 'Light'
```

Comment:
Can we change this to an object that contains both valid themes instead

Reply:
There can only be one default, so I'm not sure that makes sense. This follows suit with how `DEFAULT_CURRENCY_CODE` is declared. I *can* see the value in making it a constant to avoid littering the code with magic strings, but in this case I don't think it's gaining us much.